manifests/0000_10_config-operator_01_imagecontentsourcepolicy.crd.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: image-content-source-policy.config.openshift.io
```

Comment:
the name should be `imagecontentsourcepolicies.operator.openshift.io`

Reply:
fixed

Reply:
its still `imagecontentsourcepolicies.config.openshift.io`, it should be `imagecontentsourcepolicies.operator.openshift.io`

error from bootkube.sh 
```console
Jun 27 20:09:24 ip-10-0-11-139 bootkube.sh[1413]: "0000_10_config-operator_01_imagecontentsourcepolicy.crd.yaml": failed to create: CustomResourceDefinition.apiextensions.k8s.io "imagecontentsourcepolicies.config.openshift.io" is invalid: metadata.name: Invalid value: "imagecontentsourcepolicies.config.openshift.io": must be spec.names.plural+"."+spec.group
```

Reply:
oh, oops. Fixed now.